RocketChat/Rocket.Chat.ReactNative
# feat: add bio and nickname to profile view

## Proposed changes

Add nickname and bio to Profile. Also, I limited the input to accept only 120 characters for nickname and 260 characters for bio because the backend is limiting them.

Another PR will be introduced on the Info View.

## Issue(s)	

Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/2406
## How to test or reproduce


- Go to profile
- Edit the nickname or bio
- Submit
- Exit the profile and open it again, you should see the difference in nickname and bio
- Or see the changes on the web

## Screenshots
<img width="401" alt="Screen Shot 2023-07-31 at 16 07 06" src="https://github.com/RocketChat/Rocket.Chat.ReactNative/assets/47038980/de1ed59b-402c-4399-adc8-a8eba50d05b6">



## Types of changes




## Checklist



## Further comments

[TC-580]

[TC-580]: https://rocketchat.atlassian.net/browse/TC-580?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
